### DISCUSSION TITLE
Add sorting to list of adapters shown at teams page

### DISCUSSION THREAD

**Discussion by mcm1957** [community/community#128417] [npm]
Currently all npm packages assigned to a team inside an organization are listed in order of addition. For a big number of packages (> 100) this is not really good to read.

Please add the possibility to sort them alphabetically - either per default or by using some slection.

**Comment by ljharb** [comment]
What is an adapter, and what "teams page"?

**Comment by mcm1957** [comment]
Sorry, I referred to npm packages when writing adapter. I corrected above,

With team I refer to teams within an organization.

I would like to see a sorting at the two pages I attach as picture. Hope thats more clearly.

![image](https://github.com/npm/feedback/assets/28575778/1e1f8398-da68-4776-ba01-cb7d950dfbd6)
![image](https://github.com/npm/feedback/assets/28575778/fe0e0e53-4e7e-4fd1-97b7-0751083af7b8)


Sorry for the confusing firt posting

**Reply by ljharb** [reply]
Thanks for the clarification; in that case i super agree this would be helpful :-D
